2022-06-13 11:36:38,060 - middleware.py[line:51] - INFO: Enabled downloader middlewares:
['scrapy.downloadermiddlewares.httpauth.HttpAuthMiddleware',
 'scrapy.downloadermiddlewares.downloadtimeout.DownloadTimeoutMiddleware',
 'scrapy.downloadermiddlewares.defaultheaders.DefaultHeadersMiddleware',
 'boss.middlewares.RandomIPMiddleware',
 'scrapy.downloadermiddlewares.useragent.UserAgentMiddleware',
 'boss.middlewares.BossDownloaderMiddleware',
 'scrapy.downloadermiddlewares.retry.RetryMiddleware',
 'scrapy.downloadermiddlewares.redirect.MetaRefreshMiddleware',
 'scrapy.downloadermiddlewares.httpcompression.HttpCompressionMiddleware',
 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware',
 'scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware',
 'scrapy.downloadermiddlewares.stats.DownloaderStats']
2022-06-13 11:36:38,064 - middleware.py[line:51] - INFO: Enabled spider middlewares:
['scrapy.spidermiddlewares.httperror.HttpErrorMiddleware',
 'scrapy.spidermiddlewares.offsite.OffsiteMiddleware',
 'scrapy.spidermiddlewares.referer.RefererMiddleware',
 'scrapy.spidermiddlewares.urllength.UrlLengthMiddleware',
 'scrapy.spidermiddlewares.depth.DepthMiddleware']
2022-06-13 11:36:38,064 - middleware.py[line:51] - INFO: Enabled item pipelines:
[]
2022-06-13 11:36:38,064 - engine.py[line:313] - INFO: Spider opened
2022-06-13 11:36:38,353 - logstats.py[line:48] - INFO: Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2022-06-13 11:36:38,354 - middlewares.py[line:120] - INFO: Spider opened: jishu1
2022-06-13 11:36:38,354 - telnet.py[line:67] - INFO: Telnet console listening on 127.0.0.1:6023
2022-06-13 11:36:38,355 - middlewares.py[line:168] - INFO: 使用代理[175.173.221.78:4220]访问[https://www.zhipin.com/c100010000-p100101/?page=2]
2022-06-13 11:36:40,658 - retry.py[line:96] - DEBUG: Retrying <GET https://www.zhipin.com/c100010000-p100101/?page=2> (failed 1 times): Connection was refused by other side: 10061: 由于目标计算机积极拒绝，无法连接。.
2022-06-13 11:36:40,659 - middlewares.py[line:168] - INFO: 使用代理[123.158.99.245:4224]访问[https://www.zhipin.com/c100010000-p100101/?page=2]
2022-06-13 11:37:01,676 - retry.py[line:96] - DEBUG: Retrying <GET https://www.zhipin.com/c100010000-p100101/?page=2> (failed 2 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2022-06-13 11:37:01,677 - middlewares.py[line:168] - INFO: 使用代理[123.144.151.32:4246]访问[https://www.zhipin.com/c100010000-p100101/?page=2]
2022-06-13 11:37:01,781 - retry.py[line:118] - ERROR: Gave up retrying <GET https://www.zhipin.com/c100010000-p100101/?page=2> (failed 3 times): Could not open CONNECT tunnel with proxy 123.144.151.32:4246 [{'status': 401, 'reason': b'Authorized failed'}]
2022-06-13 11:37:01,782 - middlewares.py[line:168] - INFO: 使用代理[113.235.64.200:4243]访问[https://www.zhipin.com/c100010000-p100101/?page=2]
2022-06-13 11:37:23,169 - retry.py[line:118] - ERROR: Gave up retrying <GET https://www.zhipin.com/c100010000-p100101/?page=2> (failed 3 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2022-06-13 11:37:23,170 - middlewares.py[line:168] - INFO: 使用代理[123.144.62.135:4246]访问[https://www.zhipin.com/c100010000-p100101/?page=2]
2022-06-13 11:37:23,830 - redirect.py[line:55] - DEBUG: Redirecting (302) to <GET https://www.zhipin.com/web/common/security-check.html?seed=iUAUJwVOeRBmlbgNc76UShbQ8ea7YVwktOptymWTdqY%3D&name=a8d239c0&ts=1655091443429&callbackUrl=%2Fc100010000-p100101%2F%3Fpage%3D2&srcReferer=> from <GET https://www.zhipin.com/c100010000-p100101/?page=2>
2022-06-13 11:37:23,830 - middlewares.py[line:168] - INFO: 使用代理[42.59.100.153:4231]访问[https://www.zhipin.com/web/common/security-check.html?seed=iUAUJwVOeRBmlbgNc76UShbQ8ea7YVwktOptymWTdqY%3D&name=a8d239c0&ts=1655091443429&callbackUrl=%2Fc100010000-p100101%2F%3Fpage%3D2&srcReferer=]
2022-06-13 11:37:38,353 - logstats.py[line:48] - INFO: Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2022-06-13 11:37:44,856 - retry.py[line:118] - ERROR: Gave up retrying <GET https://www.zhipin.com/web/common/security-check.html?seed=iUAUJwVOeRBmlbgNc76UShbQ8ea7YVwktOptymWTdqY%3D&name=a8d239c0&ts=1655091443429&callbackUrl=%2Fc100010000-p100101%2F%3Fpage%3D2&srcReferer=> (failed 3 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2022-06-13 11:37:44,857 - middlewares.py[line:168] - INFO: 使用代理[123.158.99.245:4224]访问[https://www.zhipin.com/web/common/security-check.html?seed=iUAUJwVOeRBmlbgNc76UShbQ8ea7YVwktOptymWTdqY%3D&name=a8d239c0&ts=1655091443429&callbackUrl=%2Fc100010000-p100101%2F%3Fpage%3D2&srcReferer=]
